packages/react/src/components/Radio/index.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const Radio = ({
+  disabled,
+  children,
+  checked,
+  helper,
+  name,
+  value,
+  onChange,
+}) => {
+  const buildInputTag = () => (
+    <input
+      defaultChecked={checked}
```

Comment:
Is this a thing?
Shouldn't it be `checked={checked}`?

Reply:
if you want the state of the `<input>` to be managed by the `<input>` itself, then you need to pass `defaultChecked` that would be initial value of the component.
And I guess this is what we need.

If however the state of the `<input>` would be managed outside of the `<input>` (that is in `state`) then you use `checked` and then react requires to pass `onChange` hook as well.

Reply:
👍🏼